### DISCUSSION TITLE
Add “Paste” in login popup on iOS

### DISCUSSION THREAD

**Discussion by beliumgl** [ppy/osu#27828] [Ideas]
I saw a solve where I can use 3 fingers to paste, but... this impossible. When I do this, or I tap something to stop or I just tap a key on keyboard (I have very small screen). I don't know how much I will still be offline

**Comment by Walavouchey** [comment]
would you mind showing a video of the 3-finger paste gesture not working (not the password part)?

**Reply by peppy** [reply]
i can confirm it doesn't work. at some point we broke system gesture handling (probably a touch input change)

**Reply by beliumgl** [reply]
> would you mind showing a video of the 3-finger paste gesture not working (not the password part)?

It is working but not in lazer

**Reply by cdwcgt** [reply]
https://github.com/ppy/osu/assets/34775378/29021501-405c-4f21-845e-6345ca765f75

don't know why, the clipboard is empty in lazer
